modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/EffectsFragment.kt

```diff
@@ -272,26 +271,27 @@ class EffectsFragment : BaseDialogFragment(), CompoundButton.OnCheckedChangeList
         playbackManager.trackPlaybackEffectsEvent(event, props, SourceView.PLAYER_PLAYBACK_EFFECTS)
     }
 
-    @OptIn(ExperimentalCoroutinesApi::class)
     private fun FragmentEffectsBinding.setupEffectsSettingsSegmentedTabBar() {
         effectsSettingsSegmentedTabBar.setContent {
-            val podcast by viewModel.effectsLive.asFlow()
-                .mapLatest { it.podcast }
+            val podcastEffectsData by viewModel.effectsLive.asFlow()
+                .distinctUntilChanged { t1, t2 -> t1.podcast.uuid == t2.podcast.uuid && t1.podcast.playbackEffects == t2.podcast.playbackEffects }
```

Comment:
```suggestion
                .distinctUntilChanged { t1, t2 -> t1.podcast.uuid == t2.podcast.uuid && t1.podcast.playbackEffects.toData() == t2.podcast.playbackEffects.toData() }
```

This should be safer since the `PlaybackEffects` class doesn't implement `equals()` and `hashCode()`. Alternatively we could add it there. I think both options should be fine.